### DISCUSSION TITLE
How to submit multiple ShipperTrackingNumber for the same order

### DISCUSSION THREAD

**Discussion by vermouthsam** [amzn/selling-partner-api-models#3108] [General]
OrderFulfillment.xsd

**Comment by vermouthsam** [comment]
Help me

**Reply by kpconnell** [reply]
If the additional tracking number(s) are for different line item, then you send multiple messages in the XML.  If you have multiple tracking numbers for the same line item... you can send subsequent feeds, but it is really not necessary - I don't believe the amazon consumer experience support this, so the consumer would never be able to use the second tracking number anyway.  Send the first one only, or, if your shipping carrier supports a master tracking number, do that.

**Reply by bpyzikvc** [reply]
I am also looking to do a similar thing, however, I need to post additional tracking numbers to the order without specifying what item is in what package. Whenever I do this currently, the existing tracking number for the order is overwritten. What can I do?

**Reply by kpconnell** [reply]
@bpyzikvc why do you think you need to do this?  it is not supported by amazon and the consumer would not benefit from that at all.   Tracking numbers are always associated with line items.

**Reply by vermouthsam** [reply]
> @bpyzikvc为什么你认为你需要这样做？亚马逊不支持它，消费者根本不会从中受益。跟踪编号始终与行项目相关联。

Multiple products with one order number. Is it feasible to submit part of the goods when submitting the shipping order number?

**Reply by bpyzikvc** [reply]
yes, but if you send them a shipment and only include part of the goods the order will be listed as partially shipped.

```
<?xml version="1.0" encoding="UTF-8"?>
<AmazonEnvelope>
  <Header>
    <DocumentVersion>1.0</DocumentVersion>
    <MerchantIdentifier>XXXXXXX</MerchantIdentifier>
  </Header>
  <MessageType>OrderFulfillment</MessageType>
  <Message>
    <MessageID>1</MessageID>
    <OrderFulfillment>
        <AmazonOrderID>XXXXXX</AmazonOrderID>
        <FulfillmentDate>2021-08-23T20:37:00</FulfillmentDate>
        <FulfillmentData>
            <CarrierCode>UPS</CarrierCode>
            <ShippingMethod>Ground</ShippingMethod>
            <ShipperTrackingNumber>XXXXX</ShipperTrackingNumber>
        </FulfillmentData>
        <Item>
            <AmazonOrderItemCode>XXXXX</AmazonOrderItemCode>
            <Quantity>3</Quantity>
        </Item>
    </OrderFulfillment>
  </Message>
</AmazonEnvelope>
```
